cip/1.accepted/CIP2015-08-06-date-time.adoc

```diff
@@ -785,42 +835,49 @@ Components of temporal values are accessed as properties.
  * `instant.nanosecond` - the _nanosecond_ component as an integer (0-999999999)
  * `instant.timezone` - the _timezone_ as a string, either as a timezone name or as an offset from UTC in the format `±HHMM`, depending on how the timezone was specified.
  * `instant.offset` - the _timezone_ offset as a string, in the format `±HHMM`
- * `instant.epoch` - the number of milliseconds between `1970-01-01T00:00:00+0000` and the instant, positive for instants after and negative for instants before.
+ * `instant.offsetMinutes` - the _timezone_ offset in minutes
+ * `instant.offsetSeconds` - the _timezone_ offset in seconds
+ * `instant.epochMillis` - the number of milliseconds between `1970-01-01T00:00:00+0000` and the instant, positive for instants after and negative for instants before.
+ * `instant.epochSeconds` - the number of seconds between `1970-01-01T00:00:00+0000` and the instant, positive for instants after and negative for instants before.
+   For the nanosecond part of the epoch offset, the regular `nanosecond` component (`instant.nanosecond`) can be used.
 
 
 ==== Available components per instant type
 
 |===
 | *Component*  |*DateTime*|*LocalDateTime*|*Date*|*Time*|*LocalTime*
 
-|`year`        | yes | yes | yes | no  | no
-|`quarter`     | yes | yes | yes | no  | no
-|`month`       | yes | yes | yes | no  | no
-|`week`        | yes | yes | yes | no  | no
-|`weekYear`    | yes | yes | yes | no  | no
-|`day`         | yes | yes | yes | no  | no
-|`ordinalDay`  | yes | yes | yes | no  | no
-|`weekDay`     | yes | yes | yes | no  | no
-|`hour`        | yes | yes | no  | yes | yes
-|`minute`      | yes | yes | no  | yes | yes
-|`second`      | yes | yes | no  | yes | yes
-|`millisecond` | yes | yes | no  | yes | yes
-|`microsecond` | yes | yes | no  | yes | yes
-|`nanosecond`  | yes | yes | no  | yes | yes
-|`timezone`    | yes | no  | no  | yes | no
-|`offset`      | yes | no  | no  | yes | no
-|`epoch`       | yes | no  | no  | no  | no
+|`year`          | yes | yes | yes | no  | no
+|`quarter`       | yes | yes | yes | no  | no
+|`month`         | yes | yes | yes | no  | no
+|`week`          | yes | yes | yes | no  | no
+|`weekYear`      | yes | yes | yes | no  | no
+|`day`           | yes | yes | yes | no  | no
+|`ordinalDay`    | yes | yes | yes | no  | no
+|`weekDay`       | yes | yes | yes | no  | no
+|`hour`          | yes | yes | no  | yes | yes
+|`minute`        | yes | yes | no  | yes | yes
+|`second`        | yes | yes | no  | yes | yes
+|`millisecond`   | yes | yes | no  | yes | yes
+|`microsecond`   | yes | yes | no  | yes | yes
+|`nanosecond`    | yes | yes | no  | yes | yes
+|`timezone`      | yes | no  | no  | yes | no
+|`offset`        | yes | no  | no  | yes | no
+|`offsetMinutes` | yes | no  | no  | yes | no
+|`offsetSeconds` | yes | no  | no  | yes | no
```

Comment:
The property store does not even support seconds. Neither do drivers. Therefore, I think it does not make sense to have this accessor for Neo4j. Should we keep it in this proposal and document in Neo4j that in this respect we diverged from openCypher?